assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -185,3 +185,12 @@ che.keycloak.admin_username=NULL
 # from Che database. Make sense only in case ${che.keycloak.cascade_user_removal_enabled}
 # set to 'true'
 che.keycloak.admin_password=NULL
+
+# User name adjustment configuration. Represented by comma-separated key-value pairs. That is sequentially used
```

Comment:
User name adjustment configuration. Che needs to use the usernames as part of K8s object names and labels and therefore has stricter requirements on their format than the identity providers usually allow (it needs them to be DNS-compliant). The adjustment is represented by comma-separated key-value pairs. These are sequentially used as arguments to the String.replaceAll function on the original username. The keys are regular expressions, values are replacement strings that replace the characters in the username that match the regular expression.
The modified username will only be stored in the Che database and will not be advertised back to the identity provider. It is recommended to use DNS-compliant characters as replacement strings (values in the key-value pairs). Set the value of this prooperty to `NULL` to turn this feature off.
Example: `\\\\=-,@=-at-` changes `\` to `-` and `@` to `-at-` so the username `org\user@com` becomes `org-user-at-com`.